### PR TITLE
DOCS-5079 ASM Protect

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds/consolidates docs for ASM Protection

### Motivation
DOCS-5079

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hestonhoffman — COMMENTED** [review]
Looks good! Left a few suggestions

File: content/en/security/application_security/threats/protection.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  tag: "Documentation"
+  text: "Application Security Management with Datadog"
+---
+
+<div class="alert alert-info"><strong>Beta: IP blocking, user blocking, and suspicious request blocking (In-App WAF)</strong><br>
+If your service is running <a href="/agent/guide/how_remote_config_works/#enabling-remote-configuration">an Agent with Remote Configuration enabled and a tracing library version that supports it</a>, you can block attacks and attackers from the Datadog UI without additional configuration of the Agent or tracing libraries.</div>
+
+## Overview
+
+Application Security Management (ASM) Protect enables you to take blocking action to slow down attacks and attackers. Suspicious requests are blocked in real-time by the Datadog tracing libraries. Blocks are saved in the Datadog platform, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/security/application_security/threats/protection.md:16]:
```suggestion
Application Security Management (ASM) Protect enables you to slow down attacks and attackers by _blocking_ them. Suspicious requests are blocked in real-time by the Datadog tracing libraries. Blocks are saved in the Datadog platform, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services.
```

File: content/en/security/application_security/threats/protection.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+View blocked suspicious requests in the [Trace Explorer][11] by filtering on the facet `Blocked:true`.
+
+{{< img src="security/application_security/blocked-true.png" alt="ASM Trace Exporer filtered using facet Blocked set to true." style="width:100%;" >}}
+
+## Customize Protection behavior
+
+### Customize response to blocked requests
+
+{{% asm-protection-page-configuration %}}
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/security/application_security/threats/protection.md:80]: agreeed, fixed

File: content/en/security/application_security/threats/protection.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+{{< img src="/security/application_security/asm-blocking-page-html.png" alt="The page displayed as ASM blocks requests originating from blocked IPs" width="75%" >}}
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/security/application_security/threats/protection.md:82]: Agreed, fixed

File: content/en/security/application_security/threats/protection.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+From there, all ASM-protected services block incoming requests performed by the blocked IP or user, for the specified duration. All blocked traces are tagged with `security_response.block_ip` or `security_response.block_user` and displayed in the [Trace Explorer][6]. Services where ASM is disabled aren't protected.
+
+## Denylist
+
+Attackers’ IP addresses and authenticated users that are permanently or temporarily blocked are added to the _Denylist_. Manage the list on the [Denylist page][7].
+
+## Passlist
+
+You can use the _Passlist_ to permanently allow an IP address, or range of addresses, access to your application. For example, you may wish to add internal IP addresses to your passlist, or IP addresses that regularly run security audits on your application. You can also add specific paths to ensure uninterrupted access. Manage the list from the [Passlist page][8].
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/security/application_security/threats/protection.md:52]: Nice catch! fixed

**Review by hestonhoffman — APPROVED** [review]
:shipit:

**Pull request merged**